tests/group_create_mock_test.go

```diff
@@ -0,0 +1,81 @@
+package tests
+
+import (
+	"context"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/fivetran/go-fivetran"
+	"github.com/fivetran/go-fivetran/tests/mock"
+)
+
+const (
+	GroupCreateExampleResponse = `
+	{
+		"code": "Success",
+		"message": "Group has been created",
+		"data": {
+			"id": "decent_dropsy",
+			"name": "Primary_Snowflake",
+			"created_at": "2020-05-25T15:26:47.306509Z"
+		}
+	}`
```

Comment:
it's better to use fmt.Stringf to build example response using expected values as params.